packages/electron-app/src/main/app/isSubstrateRunning.ts

```diff
@@ -20,8 +20,8 @@ export async function isSubstrateRunning () {
 
   try {
     const options = {
-      headers: { 'Content-type': 'application/json' },
-      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      method: 'POST' as 'POST',
```

Comment:
nit:
`import axios, { Method } from 'axios'`
...
`method: 'POST' as Method`

^ same thing but I think this reads better

Reply:
Agreed!